### PR TITLE
CONSOLE-2361: Replace Bootstrap float utility class with PF equivalent

### PR DESCRIPTION
**Pull request by sg00dwin**
Format for "Try It" "Download YAML" buttons.

Screen Shot 2021-07-15 at 3 32 38 PM

### CODE REVIEW

**Comment by rhamilto** [comment]
Wondering if we can't maintain the layout here without adding a new partial just to float two items?

**Comment by rhamilto** [comment]
> Wondering if we can't maintain the layout here without adding a new partial just to float two items?

Seems like https://www.patternfly.org/v4/layouts/level would give us what we want here.

**Comment by sg00dwin** [comment]
Switched to use `Level` component.

**Comment by rhamilto** [comment]
Code changes look good.  Maybe some screenshots to help QE verify?

**Comment by XiyunZhao** [comment]
This PR is verified on private build and did not find any issue, the layout and view of ''Download YAML" button is not being impacted on this changes

File: frontend/public/components/sidebars/resource-sidebar-samples.tsx [hunk 1]
```diff
@@ -119,29 +124,35 @@ const ResourceSidebarSnippet: React.FC<ResourceSidebarSnippetProps> = ({
...
+        <LevelItem>
+          <Button type="button" variant="link" isInline onClick={insertSnippet}>
+            <PasteIcon className="co-icon-space-r" />
+            {t('public~Insert snippet')}
+          </Button>
+        </LevelItem>
+        <LevelItem>
+          <Button
+            type="button"
+            className="pf-u-float-right"
```
Comment (rhamilto) [inline frontend/public/components/sidebars/resource-sidebar-samples.tsx:137]: Whoops.  Looks like this was forgotten.

Comment (sg00dwin) [reply frontend/public/components/sidebars/resource-sidebar-samples.tsx]: yanked it

**Review by rhamilto — APPROVED** [review]
/lgtm

**Pull request merged**